x/emissions/keeper/genesis.go

```diff
@@ -15,9 +16,19 @@ func (k *Keeper) InitGenesis(ctx context.Context, data *state.GenesisState) erro
 
 	// ensure the module account exists
 	stakingModuleAccount := k.authKeeper.GetModuleAccount(ctx, state.AlloraStakingModuleName)
-	k.authKeeper.SetModuleAccount(ctx, stakingModuleAccount)
+	if stakingModuleAccount == nil {
```

Comment:
no. This code is wrong and got pulled in from the ORA-631 commit

this code should not be here, it needs to be deleted it will break the chain